backend/src/server.ts

```diff
@@ -15,6 +17,10 @@ app.get('/code', (_req, res) => {
 	res.send(code.gen(5, used));
 });
 
+app.post('/games/:gameId/players', (_req, res) => {
```

Comment:
As we add more routes/features it might make sense to split those into different files, but I don't think we need that now.